src/services/URLService.js

```diff
@@ -287,7 +311,7 @@ function urlService(baseUrl, mmsUrl) {
         var r = root + "/projects/" + reqOb.projectId + "/refs/" + reqOb.refId + 
             '/documents/' + reqOb.elementId + "/views";
         r = addVersion(r, reqOb.commitId);
-        return addExtended(addTicket(r), reqOb.extended);
+        return addExtended(r, reqOb.extended);
```

Comment:
I believe this is no longer used, can delete any service defs that has 'getDocumentViews' (also it's not implemented in the server)

Reply:
removed